tests/foreman/destructive/test_puppetplugin.py

```diff
@@ -56,7 +56,7 @@ def assert_puppet_status(server, expected):
             assert (f"{err_msg} '{cmd.split()[-1]}'." in str(result.stderr)) is not expected
 
 
-def test_positive_enable_disable_logic(target_sat, capsule_configured):
+def test_positive_enable_disable_logic(target_sat, subscribe_satellite, capsule_configured):
```

Comment:
Instead of `subscribe_satellite` fixture would recommend using `target_sat.register_to_cdn()` helper here

Reply:
Yeah It's also possible, I just preferred use of the fixture as it does unregister on teardown. What advantage do you see in your proposal?

Reply:
I know, but I still prefer to revert things through the fixture which committed them. Isn't the unregister via `remove-vm` workflow rather a fallback than something we should rely on?